include/kernels/gauge_wilson_flow.cuh

```diff
@@ -56,48 +56,64 @@ namespace quda
     }
   };
 
-  template <typename Arg>
-  __host__ __device__ inline auto computeStaple(const Arg &arg, const int *x, int parity, int dir)
+  template <typename Arg> struct computeStapleOpsWF {
+    using real = typename Arg::real;
+    using Link = Matrix<complex<real>, Arg::nColor>;
+    using WilsonOps = computeStapleOps;                                  // Ops for case of QUDA_GAUGE_SMEAR_WILSON_FLOW
+    using StapOp = ThreadLocalCache<Link, 0, computeStapleRectangleOps>; // offset by computeStapleRectangleOps
+    using RectOp = ThreadLocalCache<Link, 0, StapOp>;                    // offset by StapOp
+    using SymanzikOps = combineOps<computeStapleRectangleOps, KernelOps<StapOp, RectOp>>; // GAUGE_SMEAR_SYMANZIK_FLOW
+    using Ops = std::conditional_t<Arg::wflow_type == QUDA_GAUGE_SMEAR_SYMANZIK_FLOW, SymanzikOps, WilsonOps>;
```

Comment:
the use of `std::conditional_t` will break Jitify.  Can you switch this out or just `quda::conditional_t` (https://en.cppreference.com/w/cpp/types/conditional)

Reply:
Never mind, I rescind that comment, we already support `conditional_t` in Jitify.